.eslintrc.js

```diff
@@ -36,12 +36,19 @@ module.exports = {
     'arrow-body-style': ['error', 'always'],
     'import/extensions': ['error', 'always'],
     strict: ['error', 'global'],
+    'jsdoc/require-jsdoc': ['warn', {
```

Comment:
Makes sense to add this if we are adding no requirement for params / returns for private methods. 

Unfortunately params needs to use the @private tag but we can use  publicOnly: true on the required returns which is much easier.